pkg/preflight/builtin_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/replicatedhq/kurl/pkg/installer"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	yaml "gopkg.in/yaml.v3"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
why the downgrade?

Reply:
ooh because it was the only v3 usage, ok

Reply:
had to add it back anyway. unmarshal failed